rootfs/etc/profile.d/_50-workspace.sh

```
@@ -0,0 +1,29 @@
+# Files in the profile.d directory are executed by the lexicographical order of their file names.
+# This file is named _50-workspace.sh. The leading underscore is needed to ensure this file
+# executes before other files that may depend on it.
+# The number portion is to ensure proper ordering among the high-priority scripts.
+# This file depends on colors.sh, localhost.sh, and preferences.sh and must come after them
+#
+
+# file_on_host is true when the argument is a file or directory that appears to be on the Host file system.
+
+function file_on_host() {
+	local file
+	file="$(readlink -e "$1")" || return 1
```

Shellcheck (0.10.0)

[warning] file appears unused. Verify use (or export if used externally).

(SC2034)